### PR TITLE
OSDOCS-8490: TLS support for metrics ports (OCPSTRAT-970)

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.15

Issue:
[OSDOCS-8490](https://issues.redhat.com//browse/OSDOCS-8490)

Link to docs preview:
[Cluster metrics ports secured](https://69884--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-ports-use-tls)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@sunzhaohua2 this note is for [OCPCLOUD-2271](https://issues.redhat.com/browse/OCPCLOUD-2271) and [OCPCLOUD-2272](https://issues.redhat.com/browse/OCPCLOUD-2272), PTAL

**Comment by jeana-redhat** [comment]
rebased/refreshed

**Comment by gwynnemonahan** [comment]
Hey @jeana-redhat !

The links work, it's clear and concise. LGTM! 

I'm new to peer review, so @abrennan89 needs to check this, too.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -424,6 +424,12 @@ The Network Observability Operator releases updates independently from the {prod
 //[id="ocp-4-15-cluster-cloud-controller-manager-operator"]
 //=== Cloud controller managers for additional cloud providers
 
+[discrete]
+[id="ocp-4-15-ports-use-tls"]
+=== Cluster metrics ports secured
+
+With this release, the ports that serve metrics for the cluster machine approver and cloud controller manager Operators use the Transport Layer Security (TLS) protocol for additional security. (link:https://issues.redhat.com/browse/OCPCLOUD-2272[*OCPCLOUD-2272*], link:https://issues.redhat.com/browse/OCPCLOUD-2271[*OCPCLOUD-2271*])
```
Comment (abrennan89) [inline release_notes/ocp-4-15-release-notes.adoc:431]: Minor comment, but I'm curious whether "cloud controller manager Operator" etc need any capitalization or an attribute added? Probably out of scope for this PR though.

**Review by abrennan89 — COMMENTED** [review]
/lgtm

**Pull request merged**